src/storage-client/src/controller.rs

```diff
@@ -1546,6 +1546,7 @@ where
         // to the storage dependencies.
         let policies = identifiers
             .into_iter()
+            .filter(|id| self.collection(*id).is_ok())
```

Comment:
I think this snuck in from another PR.

Reply:
Unfortunately not, this function is only used for builtin migrations and silently broke at some point. Since this PR has a builtin migration it was caught.

Unfortunately, we have no good way of detecting or testing for these things without a builtin migration which is why I didn't make a separate PR.